### PR TITLE
small fixes (typo, abbreviations) + fix incomplete citations

### PR DESCRIPTION
**Pull request by marawilhelmi**
(See notion for all the requested changes that I addressed here)

## Summary by Sourcery

Fix typos, abbreviations, and incomplete citations in BibTeX entries. Adjust x-axis range and ticks in the confidence vs. performance plot. Correct a typo in the benchmark corpus figure description and fix incorrect model performance comparison in the appendix.

Bug Fixes:
- Corrected author names and formatting in multiple BibTeX entries.
- Fixed incomplete citations by adding missing fields such as 'eprint' and 'url'.
- Adjusted x-axis range and ticks in the confidence vs. performance plot to correct display issues.
- Corrected a typo in the description of the benchmark corpus figure.
- Fixed incorrect model performance comparison in the appendix.

Enhancements:
- Standardized abbreviations and expanded acronyms in the main text and BibTeX entries for clarity.

### CODE REVIEW

File: src/tex/references.bib [hunk 1]
```diff
@@ -257,12 +254,15 @@ @article{Cai_2024
   volume = {222},
 }
 
-@article{campbell2023censoring,
-  author = {Campbell, Q. L. and Herington, J. and White, A. D.},
-  date = {2023-04-20},
-  eprintclass = {cs.LG},
-  eprinttype = {arXiv},
+@misc{campbell2023censoring,
```
Comment (MrtinoRG) [inline src/tex/references.bib:257]:
```suggestion
@article{campbell2023censoring,
```

File: src/tex/references.bib [hunk 2]
```diff
@@ -450,11 +452,15 @@ @inproceedings{gemini
            Traverses},
 }
 
-@article{golovneva2024reverse,
-  author = {Golovneva, O. and Allen-Zhu, Z. and Weston, J. and Sukhbaatar, S.},
-  journaltitle = {arXiv preprint arXiv:2403.13799},
+@misc{golovneva2024reverse,
```
Comment (MrtinoRG) [inline src/tex/references.bib:455]:
```suggestion
@article{golovneva2024reverse,
```

File: src/tex/references.bib [hunk 3]
```diff
@@ -734,13 +740,15 @@ @article{miret2024llms
   date = {2024},
 }
 
-@article{moulange2023towards,
-  author = {Moulange, R. and Langenkamp, M. and Alexanian, T. and Curtis, S. and
-            Livingston, M.},
-  date = {2023-11-27},
-  eprintclass = {cs.CY},
-  eprinttype = {arXiv},
+@misc{moulange2023towards,
```
Comment (MrtinoRG) [inline src/tex/references.bib:743]:
```suggestion
@article{moulange2023towards,
```

File: src/tex/ms.tex [hunk 4]
```diff
@@ -323,7 +323,7 @@ \subsection{Model evaluation workflow}
 We used Galactica (120b)\autocite{taylor2022galactica} with the default settings.
 
 
-\subparagraph{Instruction-tuned models} In addition, we used Claude 2, Claude3 (Opus),\autocite{anthropicClaudeModelFamily2024} GPT-4,\autocite{openai2024gpt4} GPT-3.5-turbo,\autocite{brown2020language} Gemini Pro,\autocite{gemini} Mixtral-8x7b\autocite{jiang2024mixtral} LLaMA2 (7B, 13B, 70B),\autocite{touvron2023llama} as well as the 7B chat model from Perplexity.AI.
+\subparagraph{Instruction-tuned models} In addition, we used Claude 2, Claude3 (Opus),\autocite{anthropicClaudeModelFamily2024} GPT-4,\autocite{openai2024gpt4} GPT-3.5-turbo,\autocite{brown2020language} Gemini Pro,\autocite{gemini} Mixtral-8x7b,\autocite{jiang2024mixtral} LLaMA2 (7B, 13B, 70B),\autocite{touvron2023llama} as well as the 7B chat model from Perplexity.AI.
```
Comment (AdrianM0) [inline src/tex/ms.tex:326]:
```suggestion
\subparagraph{Instruction-tuned models} In addition, we tested Claude 2, Claude3 (Opus),\autocite{anthropicClaudeModelFamily2024} GPT-4,\autocite{openai2024gpt4} GPT-3.5-turbo,\autocite{brown2020language} Gemini Pro,\autocite{gemini} Mixtral-8x7b,\autocite{jiang2024mixtral} LLaMA2 (7B, 13B, 70B),\autocite{touvron2023llama} and LLaMa3 (8B, 70B, 405B).
```

**Comment by AdrianM0** [comment]
Thanks for the changes @marawilhelmi! I will try fixing the actions, and then merge

**Pull request merged**